libnet/src/ioctl.rs

```diff
@@ -617,10 +644,10 @@ pub(crate) fn delete_ipaddr(objname: impl AsRef<str>) -> Result<(), Error> {
                 (*sin6).sin6_addr.s6_addr = [0u8; 16];
             }
 
-            rioctl!(sock, sys::SIOCSLIFADDR, &ior)?;
+            rioctl_ro!(sock, sys::SIOCSLIFADDR, &mut ior)?;
```

Comment:
I'm surprised to see this one changed to `rioctl_ro` and also take an `&mut` param.

Reply:
🤦 fixed